ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -572,7 +570,7 @@ class Tab: NSObject {
       lastTitle = sessionInfo.title
       webView.interactionState = sessionInfo.interactionState
       restoring = false
-      shouldNotifyAdsServiceTabContentDidChange = false
+      isRestored = true
```

Comment:
Out of interest what is the difference between `restoring` and `isRestored`?

Reply:
`restoring` is used internally in Tab.swift during session state restoration to indicate that state restoration is in progress.
`isRestored` indicates that session state is restored.

maybe `isSessionStateRestored` and `sessionStateRestoring` naming would make more sense here? WDYT?

Reply:
Do we need two variables, can we not refactor to only use one? I have not checked usage elsewhere, but worth investigating, your thoughts?

Reply:
We can create a an enum here:
```
  enum RestorationStatus {
    case none
    case inprogress
    case restored
  }

  var restorationStatus = RestorationStatus.none
```

WDYT?

Reply:
I made this code change, but due to it touches Restoration logic, I think it is better to do it as a follow-up. WDYT?

Reply:
SGTM